installpy_standalone.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+#
+# Creates a (symlinked) python install directory for SAMADhi and sets up environment variables,
+# such that the `from cp3_llbb.SAMADhi.SAMADhi ...` imports can also be used standalone on ingrid.
+# The install directory can be configured with the PREFIX environment variable
+#
+## figure out where to install ($PREFIX/install_python, with default PREFIX=SAMDhi)
+thisscript="$(readlink -f ${0})"
```

Comment:
This doesn't work for me. I am running bash in Fedora, and ${0} evaluates to /bin/bash when the script is sourced. If the script is executed, this is fine (but then of course the export doesn't.

Reply:
So $0 are different between the bash in fedora and the one on ingrid-ui2? That's really strange... apparently I should use `BASH_SOURCE` then